src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
```

Comment:
Still not fixed 🤷 
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/571be267-6a4e-484f-a2a4-2c1539d48bc2)